spring-cloud-dataflow-common/pom.xml

```diff
@@ -18,8 +18,7 @@
 	</parent>
 
 	<properties>
-		<jayway-awaitility.version>1.7.0</jayway-awaitility.version>
```

Comment:
This managed by boot and its group has changed from `com.jayway.awaitility` to `org.awaitility`